src/com/xilinx/rapidwright/rwroute/RWRoute.java

```diff
@@ -1886,23 +1887,32 @@ protected static Design routeDesign(Design design, RWRoute router) {
     }
 
     /**
-     * The main interface of {@link RWRoute} that reads in a {@link Design} checkpoint,
-     * and parses the arguments for the {@link RWRouteConfig} object of the router.
-     * @param args An array of strings that are used to create a {@link RWRouteConfig} object for the router.
+     * The main interface of {@link RWRoute} that reads in a {@link Design} design
+     * (DCP or FPGA Interchange), and parses the arguments for the
+     * {@link RWRouteConfig} object of the router.
+     * 
+     * @param args An array of strings that are used to create a
+     *             {@link RWRouteConfig} object for the router.
      */
     public static void main(String[] args) {
         if (args.length < 2) {
-            System.out.println("USAGE: <input.dcp> <output.dcp>");
+            System.out.println("USAGE: <input.dcp|input.netlist|input.phys> <output.dcp>");
```

Comment:
I worry there's a slight implication here that you can specify just a `*.netlist`, without a companion `*.phys` which is not true. A suggestion would be to say "FPGA Interchange Physical Netlist" and have it pick up the companion Logical Netlist if one exists. 

For non-timing driven RWRoute, a logical netlist is optional, but I think there needs to be a bit more work in `PhysNetlistReader` to make it work out of the box.

Reply:
I've removed the logical netlist implication.